enterprise/api/v1/views/enterprise_customer_catalog.py

```diff
@@ -122,7 +122,7 @@ class EnterpriseCustomerCatalogViewSet(EnterpriseReadOnlyModelViewSet):
 
     USER_ID_FILTER = 'enterprise_customer__enterprise_customer_users__user_id'
     FIELDS = (
-        'uuid', 'enterprise_customer',
+        'uuid', 'title', 'enterprise_customer', 'enterprise_catalog_query',
```

Comment:
For some reason, title was already in the response.  I added it here anyway because explicit is better than implicit, especially when all the other fields are expressed.  Log this under "reasons to use drf-spectacular".